apps/etterlatte-behandling/src/main/kotlin/behandling/migrering/MigreringRoutes.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.etterlatte.behandling.omregning
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import rapidsandrivers.migrering.MigreringRequest
+import java.util.*
+
+fun Route.migreringRoutes(
+    migreringService: MigreringService
+) {
+    route("/migrering") {
+        post {
+            val request = call.receive<MigreringRequest>()
+            val behandlingId = migreringService.migrer(request)
+            call.respond(HttpStatusCode.Companion.Created, MigreringResponse(behandlingId.id, behandlingId.sak.id))
+        }
+    }
+}
+
+data class MigreringResponse(val behandlingId: UUID, val sakId: Long)
```

Comment:
Ref kommentar fra tidligere på at sakId ikke legges på rapid-eventen. Hvorfor legger vi den til i responsen om det ikke er bruk for den?

Reply:
La den på mest for å kunne skrive testen, men ser jo no at eg heller kan løyse det med nokre ekstra kall i testen.